client/html/templates/catalog/lists/items-body-standard.php

```diff
@@ -27,11 +27,28 @@
  * @category User
  * @see client/html/catalog/domains
  */
+$listTarget = $this->config( 'client/html/catalog/lists/url/target' );
+$listController = $this->config( 'client/html/catalog/lists/url/controller', 'catalog' );
+$listAction = $this->config( 'client/html/catalog/lists/url/action', 'list' );
+$listConfig = $this->config( 'client/html/catalog/lists/url/config', [] );
+$listParams = $this->get( 'listParams', [] );
 
+/** client/html/catalog/lists/infinite-scroll
+ * Enables infinite scrolling in product catalog list
+ *
+ * If set to true, products from the next page are loaded via XHR request
+ * and added to the product list when the user reaches the list bottom.
+ *
+ * @param boolean True to use infinite scrolling, false to fisable it
```

Comment:
Can you change `fisable` to `disable` and `2018.10` to `2019.10`?

Reply:
Sure. But I would love to see this feature in the 2018.x branch, additionally, if you don't mind?